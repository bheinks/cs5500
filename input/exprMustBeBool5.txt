(* 
   Expression must be of type boolean
*)

program p;
var b : boolean;
    x : integer;
begin
  b := not x;
  write('!')
end.

