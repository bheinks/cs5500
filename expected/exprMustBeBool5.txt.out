Line 9: Expression must be of type boolean
